packages/common/src/components/WideSandbox/elements.ts

```diff
@@ -19,13 +19,24 @@ export const Container = styled.div`
   overflow: hidden;
   cursor: pointer;
 
-  margin-bottom: 2rem;
```

Comment:
I believe this one needs to be back, or we need to change it in the explore page. Right now the cards in the explore page are glued to eachother.

Reply:
this file doesn0t exist anymore :/

Reply:
Added back in common